aexpect/remote.py

```diff
@@ -423,10 +428,7 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
     except Exception:
         session.close()
         raise
-    if log_filename:
-        session.set_output_func(log_function)
-        session.set_output_params((log_filename,))
-        session.set_log_file(os.path.basename(log_filename))
+    session.set_log_file(log_filename)
```

Comment:
About this line, it needs to be set after the initialization, I don't find a better way to update it. But by default, session.log_file is `None`, so even if `log_filename` is `None`, this is just a fake operation and will not affect the session.

Reply:
Yes, your suggestion is safer, I will update it in the next revision.